src/lib/utils/hooks/SynapseAPI/dataaccess/useGetDataAccessSubmission.ts

```diff
@@ -0,0 +1,39 @@
+import { useQuery, UseQueryOptions } from 'react-query'
+import {
+  mockApprovedSubmission,
+  mockSubmittedSubmission,
+  mockRejectedSubmission,
+  mockDemoSubmission,
+} from '../../../../../mocks/dataaccess/MockSubmission'
+import { delay, SynapseClientError } from '../../../SynapseClient'
+import { Submission } from '../../../synapseTypes/AccessRequirement/Submission'
+
+const submissions: Submission[] = [
+  mockSubmittedSubmission,
+  mockApprovedSubmission,
+  mockRejectedSubmission,
+  mockDemoSubmission,
+]
+
+export default function useGetDataAccessSubmission(
```

Comment:
This service doesn't exist yet (PLFM-7314)

Return a mock with the matching ID for now, so we can have design validate the component.